plugins/GSdx/GLLoader.cpp

```diff
@@ -367,7 +367,12 @@ namespace GLLoader {
 #ifdef _WIN32
 		if (status) {
 			if (fglrx_buggy_driver) {
-				fprintf(stderr, "OpenGL renderer is slow on AMD GPU due to inefficient driver. Sorry.\n");
+				fprintf(stderr, "OpenGL renderer is slow on AMD GPU due to inefficient driver.Please check the links below.\n"
```

Comment:
Better improve the whole line:
`"The \"OpenGL (Hardware)\" renderer is slow on AMD GPU's due to an inefficient driver. Check out the links below for further information.\n"`